kinetic/distribution.yaml

```diff
@@ -2257,7 +2257,12 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/ct2034/dockeros-release.git
-      version: 1.0.0-0
+      version: 1.0.2-0
+    source:
+      test_pull_requests: true
```

Comment:
I don't have direct visibility into the pr test permisions. Did you use one of these methods? http://wiki.ros.org/buildfarm/Pull%20request%20testing#Enable_pull_request_testing_for_a_specific_repository_on_Github to grant permission?

Reply:
okay, I probably don't need it now

Reply:
We'd encourage you to set it up. We just don't want the buildfarm trying to query repositories that don't have enough permissions. And unfortunately we can't automatically check for the permissions easily.

Reply:
@tfoote I have added it again in the next version #19693